todo-app/src/Task.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import TaskText from './TaskText';
+import TaskDeadLine from './TaskDeadLine';
+
+export default class Task extends React.Component {
+
+  render() {
+
+    return (
+      <li className="task-list"> 
+        <TaskText text={this.props.task.text}/>
```

Comment:
Would be nice to see some object destructuring here to prevent repetition of `this.props.task`.